sagemaker-python-sdk/tensorflow_script_mode_training_and_serving/tensorflow_script_mode_training_and_serving.ipynb

```diff
@@ -0,0 +1,254 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TensorFlow script mode training and serving\n",
+    "\n",
+    "Script Mode is a training script format for TensorFlow that lets you execute any TensorFlow training script in SageMaker with minimal modification. The [SageMaker Python SDK](https://github.com/aws/sagemaker-python-sdk) handles transferring your script to a SageMaker training instance; on the training instance, SageMaker's native TensorFlow support simply sets up some training-related environment variables and executes your training script. In this tutorial the SageMaker Python SDK is used to launch training job and deploy the trained model.\n",
+    "\n",
+    "Script Mode supports training with Python script, Python module or shell script. In this example we will use a Python script to train classification model with the [MNIST dataset](http://yann.lecun.com/exdb/mnist/). In addition this notebook also demonstrates how to perform real time inference with [SageMaker TensorFlow Serving container](https://github.com/aws/sagemaker-tensorflow-serving-container). TensorFlow Serving Container is the default inference method for Script Mode. For full documentation on TensorFlow Serving Container please visit [here](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/tensorflow/deploying_tensorflow_serving.rst).\n"
```

Comment:
* s/with Python/with a Python
* s/train/train a
* s/with the MNIST/on the MNIST
* remove "also"
* s/inference with/inference with the
* s/TensorFlow Serving Container/The TensorFlow Serving container (both times)
* commas after "In this example" and "In addition"
* comma before "please visit here"